vllm/multimodal/utils.py

```diff
@@ -191,14 +276,15 @@ def rescale_image_size(image: Image.Image,
 def try_import_video_packages() -> Any:
     try:
         import cv2
+        import decord
```

Comment:
Can you add `decord` as an optional dependency for vLLM?

Reply:
Of course, how can we make this an optional dependency? like packages in audio?

Reply:
Yep